constellations/orion/src/domains/account.go

```diff
@@ -18,6 +18,16 @@ type Account struct {
 	Password     string       `json:"password" db:"password"`
 }
 
+type AccountSum struct {
+	Id           uint         `json:"id"`
+	CreatedAt    time.Time    `json:"-" db:"created_at"`
+	UpdatedAt    time.Time    `json:"-" db:"updated_at"`
+	DeletedAt    sql.NullTime `json:"-" db:"deleted_at"`
+	PrimaryEmail string       `json:"primaryEmail" db:"primary_email"`
+	Password     string       `json:"password" db:"password"`
```

Comment:
Hmm so what I'm worried about here is whenever Account gets a new field, it would be wise to add that field here again, but it's easy to forget.

To make our life easier, have this struct only have two things:
```
type AccountSum struct {
    Account Account
    Balance int
}
```